components/line/actions/send-a-broadcast-message/send-a-broadcast-message.mjs

```diff
@@ -0,0 +1,43 @@
+import line from "../../line.app.mjs";
+
+export default {
+  name: "Send a Broadcast Message",
+  description: "Sends a broadcast message to multiple users at any time.",
+  key: "line-send-a-broadcast-message",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    line,
+    channelAccessToken: {
+      propDefinition: [
+        line,
+        "channelAccessToken",
+      ],
+    },
+    message: {
+      propDefinition: [
+        line,
+        "message",
+      ],
+    },
+    notificationDisabled: {
+      propDefinition: [
+        line,
+        "notificationDisabled",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const client = this.createLineClient(this.channelAccessToken);
```

Comment:
It may make sense to move this line and the `client.broadcast` method call to the app file as a method, and document it there with JSDocs

Reply:
> document it there with JSDocs
Can you send me the docs or a component as example ?

> It may make sense to move this line and the client.broadcast method call to the app file as a method
I figured this, but not all actions use this method or the client instance.

Reply:
Yes, take a look at https://github.com/PipedreamHQ/pipedream/blob/f366e43ded05925380971c5615c9d50144cf7cbe/components/google_sheets/google_sheets.app.mjs#L147

We like keeping these methods in the app file (even if they're only used once), since we're planning to publish the app files as their own npm packages and let users access the methods in other code. So then it's nice to have all core methods like this in the app file so they can be reused.

Reply:
I get it now!

The `createLineClient` method is already in app file, but I'm called wrong directly with `this.createLineClient`, now is correct!